spk/lua/src/service-setup.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+# Package
+PACKAGE="lua"
+DNAME="Lua"
+
+
+
+service_preinst ()
+{
+	exit 0
+}
+
+service_postinst ()
+{
+	echo "Creating startup scripts for Lua executables" >> ${INST_LOG}
+	for executable in ${SYNOPKG_PKGDEST}/bin/lua*; do
+		echo "LD_LIBRARY_PATH=/var/packages/${PACKAGE}/target/lib:\${LD_LIBRARY_PATH} $executable" > /usr/local/bin/$(basename "$executable")
```

Comment:
I would not do such wrapper scripts. Please use adequate options to generate RPATH in binaries https://en.wikipedia.org/wiki/Rpath